requirements/verification.txt

```diff
@@ -4,5 +4,8 @@ dendropy
 matplotlib
 numpy
 pandas
-scipy
+scipy==1.2 # Problems with v1.3 currently
```

Comment:
What's the issue with scipy 1.3? Be nice to have something a bit more concrete here.

Reply:
Ya good point - the issue is [here](https://github.com/statsmodels/statsmodels/issues/5747), to do with statsmodels importing a deprecated module. I'll make this a bit more descriptive.